src/serviceDesks/genesys/purecloudSetup.ts

```diff
@@ -14,7 +14,7 @@
 
 export enum setup {
   ORGANIZATION_ID = '<your-organization-id>', // https://help.mypurecloud.com/faq/how-do-i-find-my-organization-id/
-  DEPLOYMENT_ID = '<your-deployment-id>', // https://help.mypurecloud.com/articles/create-a-widget-for-web-chat/ ID of widget == DEPLOYMENT_ID
-  QUEUE_TARGET = '<your-queue-name>', // name of queue you want chat requests to go into
+  DEPLOYMENT_ID = '<your-deployment-key>', // https://help.mypurecloud.com/articles/create-a-widget-for-web-chat/ Deployment Key == DEPLOYMENT_ID
```

Comment:
these things should go to the middleware; the `src/serviceDesks/genesys` is a client side code and should not know anything about server side set up/code

Reply:
These variables don't actually interact with the server code. These are used in the serviceDesk code only as part of messaging, which doesn't require the middleware server to be run.